src/geo/gmaps/gmaps-cartodb-layer-group-view.js

```diff
@@ -285,11 +293,12 @@ _.extend(
     },
 
     _refreshView: function () {
-      var overlayIndex = this.gmapsMap.overlayMapTypes.getArray().indexOf(this);
+      var overlays = this.gmapsMap.overlayMapTypes;
+      var overlayIndex = this._getOverlayIndex();
+
       if (overlayIndex >= 0) {
-        this.gmapsMap.overlayMapTypes.removeAt(overlayIndex);
+        overlays.setAt(overlayIndex, overlays.getAt(overlayIndex));
```

Comment:
This seems a bit strange since we're setting the overlay at the same exact place. I guess that `setAt` triggers something internally in Google Maps. If not, this has no much sense.

Reply:
It's basically a reload, before we were doing `remoteAt 0` -> `setAt 0`, but just doing `setAt` works fine.